pages/sign-up.js

```diff
@@ -95,12 +99,8 @@ const SignUp = () => {
                         <div className="flex items-center mt-3 mb-6">
                           <Checkbox
                             name="agreeContact"
-                            type="checkbox"
                             label={
-                              <label
-                                htmlFor="remember-me"
-                                className="block ml-2 text-[0.75rem] text-[#344055] font-normal font-Inter "
-                              >
+                              <label>
```

Comment:
label, not clickable apply checked on the checkbox but work for the checkbox group kindly help review it also

Reply:
This is because you didn't set `value` here (check the `Checkbox` component on line 100). 

The `htmlFor` code is based on the value of `value` so if it is not defined it will cause unexpected output because of the value of `id` is gotten from `value`.

```js
        <input
          id={`options-${value}`}
```

Reply:
```js

        <input
          id={`options-${name}-${value.toString()}`}

```